guestbook.json

```diff
@@ -9,5 +9,15 @@
     "isRoomClean" : "no",
     "bill" : "0.00",
     "request" : "Apply for room service"
+  }, {
+    "name" : "Juuu",
+    "phone" : "98765432",
+    "email" : "johnd@example.com",
+    "room" : "05-73",
+    "dateRange" : "13/09/22 - 15/09/22",
+    "numberOfGuests" : "1",
+    "isRoomClean" : "yes",
+    "bill" : "0.00",
+    "request" : "-"
```

Comment:
What is this entry for?

Reply:
This was a new entry I created when I was manually testing the edit command for isRoomClean.